R/plot_sankey.R

```diff
@@ -0,0 +1,85 @@
+#' Create a sankey plot
+#'
+#' @param data A data frame like [toy_data].
+#'
+#' @return
+#' @export
+#'
+#' @examples
+#' plot_sankey(toy_data)
+plot_sankey <- function(data) {
+  data_links <- data |>
+    mutate(
+      source = "bank",
+      target = "PCTR_risk_category",
+      value = "amount",
+      middle_node2 = "tilt_sec"
```

Comment:
@maurolepore, if there are " " added, the new columns takes the string as values added in the column. For example, middlenode2 takes "tilt_sec" as values instead of "sector_a" and "sector_b "(which is what I want) - I thought that [dplyr did not need " " anymore in the mutate() function ?](https://dplyr.tidyverse.org/reference/mutate.html)

Reply:
I'll try to find the relavant article and share it later. But note the behavour is  different for `mutate()` and `filter()` compared to `select()`, and `rename()`. 

For mutate you can use a bare name like `my_col` or `.data$my_col` while for `select()` you now use "my_col".

Reply:
Okay nice ! In those lines of code it is a mutate() so I will change it into the bare name. Thanks @maurolepore.